jsapp/js/components/activity/activityLogs.query.ts

```diff
@@ -1,9 +1,10 @@
 import {keepPreviousData, useQuery} from '@tanstack/react-query';
 import type {FailResponse, PaginatedResponse} from 'js/dataInterface';
 import {
-  AUDIT_ACTION_TYPES,
+  AUDIT_ACTION_TYPES} from './activity.constants';
```

Comment:
This should be a single line :)

Reply:
Formatting didn't apply for some reason! Thanks for noticing.